vendor/deps/cli-kit/lib/cli/kit/system.rb

```diff
@@ -163,6 +163,15 @@ def split_partial_characters(data)
           [data.byteslice(0...partial_character_index), data.byteslice(partial_character_index..-1)]
         end
 
+        def os
+          require 'rbconfig'
+
+          host = RbConfig::CONFIG["host"]
+          return :mac if /darwin/.match(host)
+          return :linux if /linux/.match(host)
+          :windows if /mingw32/.match(host)
```

Comment:
I think this should be more explicit, since if it's not windows we're returning `nil`
```
   return :windows if /mingw32/.match(host)
   nil
end
```
(same goes for [this](https://github.com/Shopify/shopify-app-cli/pull/775/files#diff-8cbd18e0c5bee5819900f6ac4f20124cR228) one)